tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Suggest providing more details on the libraries:</p><ul><li><a href="https://pandas.pydata.org/" target="_blank"><strong>pandas</strong></a>: data analysis (including the <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html" target="_blank">DataFrame</a> data structure)</li><li><a href="https://altair-viz.github.io/" target="_blank"><strong>altair</strong></a>: declarative visualization library (used for charting)</li><li><a href="https://numpy.org/" target="_blank"><strong>numpy</strong></a>: array-processing package (used for linear regression)</li><li><a href="https://python-visualization.github.io/folium/" target="_blank"><strong>folium</strong></a>: interactive web map</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done.</p>